src/client.ts

```diff
@@ -352,6 +362,23 @@ export class SubscriptionClient {
     }
   }
 
+  private clearInactivityTimeout() {
+    if (this.inactivityTimeoutId) {
+      clearTimeout(this.inactivityTimeoutId);
+      this.inactivityTimeoutId = null;
+    }
+  }
+
+  private setInactivityTimeout() {
+    if (this.inactivityTimeout > 0 && this.activeSubscriptions === 0) {
+      this.inactivityTimeoutId = setTimeout(() => {
+        if (this.activeSubscriptions === 0) {
+          this.close();
```

Comment:
Should add a test to verify that if the connection is closed due to inactivity, a future subscribe will correctly reopen the connection.

Reply:
Thanks, added the relevant test case.